### PR TITLE
move `beforeinput` event and `input` event from `HTMLElement` to `Element`

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

in UI Events, the `beforeinput` event and `input` event has failed link, this is because the two events' documentation is only written under `HTMLElement` not under `Element`

and other event types (like mouse events) in the standard that target to element do documented in `Element` rather then `HTMLElement`

(also, interstingly, as I check via event handle property (like `oninput`), they are available on `HTMLElement` `SVGElement` or `MathMLElement`, and `Window` or `Document`, but not available on `Element`, this also happens to other *UI Events*, this behavior is similar to *Pointer Events* and *Touch Events*)

image
---

also add `Element: auxclick` and `Element: contextmenu` into group data of UI Events, see https://w3c.github.io/uievents/

### CODE REVIEW

**Comment by bsmth** [comment]
Thanks @skyclouds2001 - taking a look shortly

File: files/en-us/web/api/ui_events/index.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ The UI Events API defines a system for handling user interactions such as mouse
   - : Fired when loading a resource has been aborted (for example, because the user canceled it).
 - {{domxref("Element.auxclick_event", "auxclick")}}
   - : Fired when the user clicks the non-primary pointer button.
-- {{domxref("Element.beforeinput_event", "beforeinput")}}
+- {{domxref("HTMLElement.beforeinput_event", "beforeinput")}}
```
Comment (bsmth) [inline files/en-us/web/api/ui_events/index.md:44]: There's two more on line 200 (`beforeinput_event` and `input_event`)

**Comment by bsmth** [comment]
Searching through other content brings up these:

https://github.com/mdn/content/blob/00a2f5f6cc67c3c95247a43bb8bbfd6c5b2391d6/files/en-us/web/svg/attribute/end/index.md?plain=1#L70-L71

Could we fix it in this PR, also? Those are the last ones I can find in other docs.

**Comment by skyclouds2001** [comment]
honestly, I check the standard again, and notice that other event types that target to element do implements in `Element` rather then `HTMLElement`

so i guess the better solution is move `input` event and `beforeinput` event from `Element` from `HTMLElement`

**Comment by queengooborg** [comment]
BCD places events based upon where the events are defined in the spec, rather than the event handlers -- so, this change makes sense to me, yes.

**Comment by bsmth** [comment]
Looking great, and I see the corresponding BCD is done, so I think we can merge shortly. Is it correct to say there are some minor leftovers of (`HTMLElement/input_event` & `HTMLElement/beforeinput`) in the following two docs:

- __/files/en-us/web/api/document/execcommand/index.md__ ([link](https://github.com/mdn/content/blob/13d979ec8bc1daf315fc6a17e38cb855cf2e4ef1/files/en-us/web/api/document/execcommand/index.md?plain=1#L19))
  - ``[`beforeinput`](/en-US/docs/Web/API/HTMLElement/beforeinput_event) and [`input`](/en-US/docs/Web/API/HTMLElement/input_event)``: 
- __/files/en-us/web/api/element/beforeinput_event/index.md__
  -  `... in contrast to the {{domxref("HTMLElement/input_event", "input")}} event,`

**Review by bsmth — APPROVED** [review]
After the BCD has landed in https://github.com/mdn/browser-compat-data/pull/21640/, leaving +1 with some comments for flaws that we can fix here. We can merge after cleaning these up if you like

Thank you!

**Comment by skyclouds2001** [comment]
> * HTMLElement/beforeinput_event

fixed!

**Comment by bsmth** [comment]
Looking great, let's merge 🚢

**Pull request merged**